web/pdf_viewer.js

```diff
@@ -846,7 +846,9 @@ class PDFViewer {
             this.#annotationEditorUIManager = new AnnotationEditorUIManager(
               this.container,
               this.eventBus,
-              pdfDocument?.annotationStorage
+              pdfDocument?.annotationStorage,
+              pdfDocument.filterFactory,
```

Comment:
How about we simply pass in the `pdfDocument` reference instead now, and let the Editor access the properties/methods it needs?